HomeWork_20/HW_20.py

```diff
@@ -0,0 +1,8 @@
+
```

Comment:
You have removed a line here, but I was saying that you can remove the whole file